packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -181,9 +185,19 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
     registerCommands(registry: CommandRegistry): void {
         super.registerCommands(registry);
         registry.registerCommand(FileNavigatorCommands.REVEAL_IN_NAVIGATOR, {
-            execute: () => this.openView({ activate: true }).then(() => this.selectWidgetFileNode(this.shell.currentWidget)),
-            isEnabled: () => Navigatable.is(this.shell.currentWidget),
-            isVisible: () => Navigatable.is(this.shell.currentWidget)
+            execute: (event?: Event) => {
+                const widget = this.getTargetedWidget(event);
+                this.selectWidgetFileNode(widget || this.shell.currentWidget);
```

Comment:
It does not look correct we should select a node if the navigator fail to open, please align with the previous semantic

Reply:
@akosyakov thank you for ur review, using the previous semantic it reveals the current active widget. Is there a way that could be fixed with the previous semantic?